### PR TITLE
Nurmash

### PR DESCRIPTION
**Pull request by Safrus**

### CODE REVIEW

File: app/src/main/java/com/example/myapptest/Sotries.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package com.example.myapptest;
+
+public class Sotries {
```
Comment (Safrus) [inline app/src/main/java/com/example/myapptest/Sotries.java:3]: Brother, change this f**king Sotries to Stories) Thanks!

File: app/src/main/java/com/example/myapptest/Stories.java [hunk 2]
```diff
@@ -1,6 +1,6 @@
 package com.example.myapptest;
 
-public class Sotries {
+public class Stories {
```
Comment (Safrus) [inline app/src/main/java/com/example/myapptest/Stories.java:4]: What the f**k bro,are you playing games with me? where is indfsfsdfsfdsfsdf

Comment (DamiNo007) [reply app/src/main/java/com/example/myapptest/Stories.java:4]: Fuck off!

Comment (DamiNo007) [reply app/src/main/java/com/example/myapptest/Stories.java:4]: Wait a second!

Comment (DamiNo007) [reply app/src/main/java/com/example/myapptest/Stories.java:4]: Wait few more seconds!

Comment (Safrus) [reply app/src/main/java/com/example/myapptest/Stories.java:4]: okey...

Comment (DamiNo007) [reply app/src/main/java/com/example/myapptest/Stories.java:4]: Just WAIT!

**Pull request merged**